.husky/commit-msg

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-BRANCH_NAME=$(git branch | grep '*' | sed 's/* //')
+BRANCH_NAME=$(git branch | grep '*' | sed 's/.*\*.* //')
```

Comment:
I think it should be: `git branch --no-color`

Reply:
Nice one @kare